src/pyroboplan/ik/differential_ik.py

```diff
@@ -54,6 +55,9 @@ def __init__(
             max_step_size : float
                 Maximum gradient step size, between 0 and 1, based on ratio of current distance to target to initial distance to target.
                 To use a fixed step size, set both minimum and maximum values to be equal.
+            joint_weights : list[float], optional
+                A list of relative weights for different joints, used in computing the Jacobian pseudoinverse.
```

Comment:
Might be worth adding some clarification on high vs low weights and how it impacts the solver. When I first went through this I inverted... this was the only bit that confused me here.